File: backend/src/main/java/ipeps/pwd/wallet/entity/payload/EmployeeUpdatePayload.java
package ipeps.pwd.wallet.entity.payload;


import ipeps.pwd.wallet.entity.Company;
import ipeps.pwd.wallet.entity.enums.EmployeeStatus;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.validation.constraints.NotNull;
import java.util.UUID;


@Data
@NoArgsConstructor
@AllArgsConstructor
public class EmployeeUpdatePayload {
    UUID Employee_id;
    String Lastname;
    String Firstname;
    String Address;
    String Gender;
    String Birthday;
    String Ssin;
    EmployeeStatus status;
    Company company;
    Boolean active;
    Boolean deleted;
    Boolean deleted_by;
    String City;
    String Phone;
    String Email;
    String Picture;
}

File: WalletApp/src/app/modules/employee/payload/EmployeeUpdatePayload.ts
import {Company} from "../../company/model/business/company";
import {EmployeeStatus} from "../model/enum/EmployeeStatus";

export interface  EmployeeUpdatePayload{
  Employee_id: string;
  Lastname : string;
  Firstname: string;
  Address: string;
  Gender: string;
  Birthday: string;
  Ssin: string;
  status : EmployeeStatus;
  company: Company;
  active : boolean;
  picture: string;
}
